ci_watson/scripts/okify_regtests.py

```diff
@@ -196,39 +267,58 @@ def pushd(newdir: os.PathLike):
 
 def main():
     parser = ArgumentParser(
-        description='"okifies" a set of failing regression test results, by overwriting '
+        description='"Okifies" a set of failing regression test results, by overwriting '
         "truth files on Artifactory so that a set of failing regression test results becomes correct. "
-        "Requires JFrog CLI (https://jfrog.com/getcli/) configured with credentials (`jf login`) "
-        "and write access to the desired truth file repository (`jwst-pipeline`, `roman-pipeline`, etc.)."
+        "Requires JFrog CLI (https://jfrog.com/getcli/) configured with credentials (jf login) "
+        "and write access to the desired truth file repository (jwst-pipeline, roman-pipeline, etc.)."
     )
-    (
-        parser.add_argument(
-            "observatory",
-            type=Observatory,
-            choices=list(Observatory),
-            help="Observatory to overwrite truth files for on Artifactory",
-        ),
+    parser.add_argument(
+        "observatory",
+        type=Observatory,
+        choices=list(Observatory),
+        help="Observatory to overwrite truth files for on Artifactory.",
     )
     parser.add_argument(
         "run_number",
-        help="GitHub Actions job number of regression test run (see https://github.com/spacetelescope/RegressionTests/actions)",
+        help=("GitHub Actions job number of regression test run (see "
+              "https://github.com/spacetelescope/RegressionTests/actions)."),
         metavar="run-number",
     )
+    parser.add_argument(
+        "--version",
+        action="version",
+        version=f"ci-watson {ci_watson.__version__}",
+        help="Print package version and exit.",
+    )
     parser.add_argument(
         "--dry-run",
         action="store_true",
-        help="do nothing (passes the `--dry-run` flag to JFrog CLI)",
+        help="Do nothing (passes the --dry-run flag to JFrog CLI).",
+    )
+    parser.add_argument(
+        "--dlpath",
+        type=str,
```

Comment:
```suggestion
        type=Path,
```

Reply:
What should be the default if you want to enforce Path object here?

Reply:
I am actually going to push back on this suggestion because it complicates the checks. All `tmp_path` is used for is input into `pushd` that does not actually need a `Path` input (so I would argue that the typing is inaccurate). `str` input would work just as well and is simpler.

Reply:
that's fair, and since the docs indicate that it should be a path string this should be fine